softicar/platform
# PLAT-1239 Workflow Version Runnable Validation

Test Plan:
- export workflow version
- import workflow version (without change)
- observer import and activation works
- modify exported version to refer to non-existing UUIDs
- import modified workflow version
- observe import works but activation fails with proper error message

![workflow-validation](https://user-images.githubusercontent.com/83839745/210749019-573bb2de-63c0-49b7-b146-f3dcc4b5eda3.gif)
